src/nibetaseries/interfaces/nistats.py

```diff
@@ -59,11 +59,11 @@ def _run_interface(self, runtime):
         else:
             confounds = None
 
-        # low_pass, switch from Hz to Period
-        if self.inputs.low_pass:
-            low_pass_period = int(1 / self.inputs.low_pass)
+        # high_pass, switch from Hz to Period
+        if self.inputs.high_pass:
+            high_pass_period = int(1 / self.inputs.high_pass)
         else:
-            low_pass_period = 128
+            high_pass_period = 128
```

Comment:
This default is kind of hidden. @jdkent would you mind if I added a default to the workflow and CLI instead?

Reply:
yes, I agree that would be better. As a note, this is the [default used for nistats](https://github.com/nistats/nistats/blob/4be29274fae131b36052c7f1c087b10f7deec519/nistats/first_level_model.py#L264) as well.